Modules/Role/Http/Services/RoleService.php

```diff
@@ -43,32 +39,24 @@ public function submitRole(array $requestData): array
      */
     public function show(): array
     {
-        $role = Role::where('role', '!=', 'admin')
-            ->where('role', '!=', 'Content Moderator')
+        $role = Role::where('id', '!=', '1')
```

Comment:
Also if there are more than one roles, change variable name to $roles.

Reply:
Fixed